cmd/root.go

```diff
@@ -1,30 +1,41 @@
 package cmd
 
-import "github.com/spf13/cobra"
+import (
+	"github.com/mgutz/ansi"
+	"github.com/spf13/cobra"
+)
 
 func New() *cobra.Command {
 
+	chartCommand := newChartCommand()
+
 	cmd := &cobra.Command{
 		Use:   "diff",
 		Short: "Show manifest differences",
+		//Alias root command to chart subcommand
+		Args: chartCommand.Args,
+		// parse the flags and check for actions like suppress-secrets, no-colors
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			if nc, _ := cmd.Flags().GetBool("no-color"); nc {
+				ansi.DisableColors(true)
+			}
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.Println(`Command "helm diff" is deprecated, use "helm diff upgrade" instead`)
+			return chartCommand.RunE(cmd, args)
+		},
 	}
 
-	chartCommand := newChartCommand()
```

Comment:
Huh, why remove all this? The aliasing is done to keep backward compatibility.  I would like to keep that for a while to make that this doesn't break existing CI systems.

Reply:
oops...!! I didn't remove it.  It is there, just fitted it inside, `cobra.command`
```go
	chartCommand := newChartCommand()

	cmd := &cobra.Command{
		Use:   "diff",
		Short: "Show manifest differences",
		//Alias root command to chart subcommand
		Args: chartCommand.Args,
```

Reply:
does this need changes/reverting, its exactly the same code..!!

Reply:
Ah, ok my Mistake, thanks for clearing this up.

Reply:
removed it